packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/mouseUtils/handleMouseDownEvent.ts

```diff
@@ -53,65 +53,99 @@ export function handleMouseDownEvent(
                     selection?.setBaseAndExtent(state.firstTarget, 0, state.lastTarget, 0);
                     selectTable(editor, state);
                 }
-
                 return;
             }
         }
     }
-    if (which == LEFT_CLICK && !shiftKey) {
-        clearState(state, editor);
+    if (which == LEFT_CLICK) {
+        // Triple clicking a cell will select that cell
+        if (detail >= 3) {
+            editor.runAsync(editor => {
+                const sel = editor.getDocument().defaultView?.getSelection();
+                const first = getCellAtCursor(editor, sel?.anchorNode);
+                const firstTable = getTableAtCursor(editor, first);
+                const targetTable = getTableAtCursor(editor, first);
+                if (firstTable! == targetTable! && safeInstanceOf(first, 'HTMLTableCellElement')) {
+                    state.vTable = new VTable(first);
+                    const firstCord = getCellCoordinates(state.vTable, first);
+
+                    if (!firstCord) {
+                        return;
+                    }
+
+                    state.vTable.selection = {
+                        firstCell: firstCord,
+                        lastCell: firstCord,
+                    };
+
+                    state.firstTarget = first;
+                    state.lastTarget = first;
+                    selectTable(editor, state);
 
-        if (getTableAtCursor(editor, event.rawEvent.target)) {
-            const doc = editor.getDocument() || document;
+                    state.tableSelection = true;
+                    state.firstTable = firstTable as HTMLTableElement;
+                    state.targetTable = targetTable;
+                    updateSelection(editor, first, 0);
```

Comment:
Curious, can this be replaced by calling editor.select({... (a table selection rangeEx) })?

I understand the original code is also like this, but just want to try improve it

Reply:
For triple click I think editor.select should be enough. For the other scenarios I suggest testing if the selection is not broken. since setting selection with range does not mantain the initial and the last node selected.

Reply:
I tried using editor.select but it does not seem to work as expected.